modtree/modtree
# feat: UI setup for adding a new graph

### Summary of changes

- Create non-functional `add-new` graph page
- Create semi-functional dropdown graph picker on `main` graph page

### Testing

Add new page

- Create a 2nd degree in the frontend
- Try to toggle between degrees on the add graph page
- You should see the placed modules comment change

Graph picker

- Create a 2nd graph using `yarn gc` in postman
- Try to toggle between graphs on the main graph page
- The chosen graph should persist even after a refresh. The displayed graph however is not updated at the moment.
